app/views/documents/edit.html.erb

```diff
@@ -20,10 +20,10 @@
       id: "title",
       html_for: "title-field"
     }
-    if I18n.exists?("document_types.#{@edition.document_type.id}.fields.title.guidance")
+    if t_doctype_key_exists?("document_types.#{@edition.document_type.id}.fields.title.guidance")
       title_contextual_guidance.merge!({
-        title: t("document_types.#{@edition.document_type.id}.fields.title.guidance.title"),
-        content: t("document_types.#{@edition.document_type.id}.fields.title.guidance.body_govspeak"),
+        title: t_doctype("document_types.#{@edition.document_type.id}.fields.title.guidance.title"),
```

Comment:
We should avoid the interpolation here, since we repeat it a lot. We could think about having multiple arguments to the helper, and maybe even having a separate one for fields.

Reply:
I considered this, but couldn't decide on a way forward that was much better than the original. We need to pass the document type ID, the field name and then the name of the guidance we want to retrieve, so would end up with something like this:

```diff
- t_doctype("document_types.#{@edition.document_type.id}.fields.title.guidance.title"),
+ t_doctype_field_guidance(@edition.document_type.id, "title", "title"),
```

This felt like one of those cases where some ugly long strings were cleaner than a poor abstraction, but I'll happily go in this direction if you think otherwise.

Another thing I considered was some form of on-the-fly object-orientation, e.g.:

```diff
- t_doctype("document_types.#{@edition.document_type.id}.fields.title.guidance.title"),
+ DocumentTypeHelper.new(@edition.document_type).guidance_for("title").title,
```

This reads cleaner but doesn't map well with the 'helper module' approach we've gone for - it feels more like a presenter. We don't yet have any presenters in Content Publisher and I didn't want to establish a precedent.

Reply:
Talked IRL, decided on this as a way forward:

```
t_doctype(@edition, "fields.title.guidance.title"),
```

This will only appear in `app/views/documents/fields/_govspeak_input.html.erb`, as we will no longer need to use the helper in `app/views/documents/edit.html.erb` (we'll use I18n directly).

Reply:
Now done 👍